os-appium-tests/awsConfiguration.yml

```diff
@@ -0,0 +1,84 @@
+version: 0.1
```

Comment:
This file might be outdated and there are differences between Android and iOS runs. We should revisit this later, or remove it at all, since we are using saucelabs.

Reply:
This is relevant for the boilerplate repository as well.

This file is present in the boilerplate repository so it should also be reviewed there. As for the other 2 files they are in the framework project (and not the boilerplate) so they won't need to be reviewed there.